### DISCUSSION TITLE
Contributors list showing wrong accounts in repository

### DISCUSSION THREAD

**Discussion by TeoNikolov** [community/community#27809] [New to GitHub] [answered]
I have a repository (https://github.com/TeoNikolov/group-project/) that has a contributor with no commits in the repo. In fact, this was an account that I used before I started using my work one, so when I transferred the repository, the old account remained in the contributor list shown on the repo page (not shown in the contributors through settings). The issue is the same as https://github.com/orgs/community/discussions/23966 .

Any way I can fix it without having to remake the repo from scratch (which will be a pain to do)?

Cheers!

**Comment by jfullstackdev** [comment]
try blocking him 

> The user is removed as a collaborator from all the projects and classic projects owned by you

https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account#about-blocking-users

**Reply by TeoNikolov** [reply]
I did that just now, but no change yet. Maybe it will take some time for the change to take effect, but my suspicion is that it will not help since the account is not an actual collaborator in the repository. They only appear on the list on the repo page, so it's most likely outdated information on GitHub side.

**Comment by airtower-luna** [comment, marked as answer]
In that other thread you linked the solution was that a staff member had the contributors list rebuilt. So your best option is probably to [create a support ticket](https://support.github.com/request) and ask them to do that.

**Reply by TeoNikolov** [reply]
I believe you are correct. I will write the support staff, and come back to resolve the issue when they confirm it.

Edit: A support ticket is the route to go. It was quick, and effective! :)
